aggregation/types_options.go

```diff
@@ -540,4 +365,5 @@ func defaultTimerQuantileTypeStringFn(quantile float64) []byte {
 }
 
 func noopTransformFn(b []byte) []byte   { return b }
```

Comment:
Do we still need the noop type?

Reply:
It's the default transformation function, I think query uses it to infer aggregation type? Also might be useful for OSS users.